src/CodeStyle/CSharp/CodeFixes/CSharpFormattingAnalyzerImpl.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.CodingConventions;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal class CSharpFormattingAnalyzerImpl : AbstractFormattingAnalyzerImpl
+    {
+        private readonly EditorConfigOptionsApplier _editorConfigOptionsApplier = new EditorConfigOptionsApplier();
+
+        public CSharpFormattingAnalyzerImpl(DiagnosticDescriptor descriptor)
+            : base(descriptor)
+        {
+        }
+
+        protected override OptionSet ApplyFormattingOptions(OptionSet optionSet, ICodingConventionContext codingConventionContext)
+        {
+            return _editorConfigOptionsApplier.ApplyConventions(optionSet, codingConventionContext.CurrentConventions, LanguageNames.CSharp);
```

Comment:
is the VB version of this actually different (beyond the language?)?  If not, should this just move up?

Reply:
➡️ Yes, the options type(s) are different, and for C# are located in assemblies that are not referenced by the base class.